lib/mvi/responses/id_parser.rb

```diff
@@ -3,36 +3,44 @@
-      #              H = deprecated(do not use)
-      #              L = pending local merge (do not use)
-      #              PCE = ??
+      #           EI - employee identifier
+      #           PN = patient number
+
+      # id status -  A = Active (only applies to correlation IDs)
+      #              P = Permanent (only applies to ICNs)
+
+      # Should definitely NOT be using id_status
+      #              H = Deprecated due to local merge
+      #              D = Deprecated from a Duplicate
+      #              M = Deprecated from a Mismatch
+      #              U - Deprecated from an Unlink
+      #              L = pending local merge - This is a value identified for correlations including EDIPI when the
+      #                   identifier has been deprecated/associated to another active ID.
+      #              PCE = Pending Cat Edit correlations (unsure if this should be used, likely not)
 
       def parse(ids)
         ids = ids.map(&:attributes)
-
         {
-          icn: select_ids(select_extension(ids, ICN_REGEX, CORRELATION_ROOT_ID))&.first,
-          sec_id: select_ids(select_extension(ids, /^\w+\^PN\^200PROV\^USDVA\^\w+$/, CORRELATION_ROOT_ID))&.first,
-          mhv_ids: select_ids(select_extension(ids, /^\w+\^PI\^200MH.{0,1}\^\w+\^\w+$/, CORRELATION_ROOT_ID)),
-          active_mhv_ids: select_ids(select_extension(ids, /^\w+\^PI\^200MH.{0,1}\^\w+\^A$/, CORRELATION_ROOT_ID)),
-          edipi: select_ids(select_extension(ids, /^\w+\^NI\^200DOD\^USDOD\^\w+$/, EDIPI_ROOT_ID))&.first,
-          vba_corp_id: select_ids_except(select_extension(ids, /^\w+\^PI\^200CORP\^USVBA\^\w+$/, CORRELATION_ROOT_ID),
-                                         %w[H L])&.first,
-          vha_facility_ids: select_facilities(select_extension(ids, /^\w+\^PI\^\w+\^USVHA\^\w+$/, CORRELATION_ROOT_ID)),
-          birls_id: select_ids(select_extension(ids, /^\w+\^PI\^200BRLS\^USVBA\^\w+$/, CORRELATION_ROOT_ID))&.first,
-          vet360_id: select_ids(select_extension(ids, /^\w+\^PI\^200VETS\^USDVA\^A$/, CORRELATION_ROOT_ID))&.first,
-          icn_with_aaid: ICNWithAAIDParser.new(full_icn_with_aaid(ids)).without_id_status
+          icn:              select_ids(select_extension(ids, PERMANENT_ICN_REGEX,                 VA_ROOT_OID))&.first,
+          sec_id:           select_ids(select_extension(ids, /^\w+\^PN\^200PROV\^USDVA\^A$/,      VA_ROOT_OID))&.first,
+          mhv_ids:          select_ids(select_extension(ids, /^\w+\^PI\^200MH.{0,1}\^\w+\^\w+$/,  VA_ROOT_OID)),
+          active_mhv_ids:   select_ids(select_extension(ids, /^\w+\^PI\^200MH.{0,1}\^\w+\^A$/,    VA_ROOT_OID)),
+          edipi:            select_ids(select_extension(ids, /^\w+\^NI\^200DOD\^USDOD\^A$/,       DOD_ROOT_OID))&.first,
+          vba_corp_id:      select_ids(select_extension(ids, /^\w+\^PI\^200CORP\^USVBA\^A$/,      VA_ROOT_OID))&.first,
+          vha_facility_ids: select_facilities(select_extension(ids, /^\w+\^PI\^\w+\^USVHA\^\w+$/, VA_ROOT_OID)),
```

Comment:
Did _not_ change regex to require 'A'. 
Presence of IDs is checked [here](https://github.com/department-of-veterans-affairs/vets-api/blob/91b76792905302a5eb5eb2b4d7661050af54dd89/app/models/user.rb#L182) to determine if a user is a VA patient. 
The individual IDs themselves are not used anywhere AFAICT.